src/components/structures/login/Registration.js

```diff
@@ -223,13 +217,13 @@ module.exports = React.createClass({
             this.props.onLoggedIn({
                 userId: response.user_id,
                 deviceId: response.device_id,
-                homeserverUrl: this.registerLogic.getHomeserverUrl(),
-                identityServerUrl: this.registerLogic.getIdentityServerUrl(),
+                homeserverUrl: this._matrixClient.getHomeserverUrl(),
+                identityServerUrl: this._matrixClient.getIdentityServerUrl(),
                 accessToken: response.access_token
             }, teamToken);
         }).then(() => {
             return this._setupPushers();
-        }).done();
```

Comment:
The promise returned from this is not a q promise, and so does not have a .done, as it turns out. :/